bin/travis-build.bash

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
+set -e
+echo "This is travis-build.bash..."
+
+echo "-----------------------------------------------------------------"
+echo "Installing the packages that CKAN requires..."
+sudo apt-get update -qq
+sudo apt-get install solr-jetty libcommons-fileupload-java libpq-dev postgresql postgresql-contrib python-lxml postgresql-9.3-postgis-2.1
+
+# if [ $CKANVERSION == '2.8' ]
+# then
+# 	sudo apt-get install postgresql-9.3-postgis-2.1
+# elif [ $CKANVERSION == '2.3' ]
+# then
+# 	sudo apt-get install postgresql-9.1-postgis
+# fi
+
+pip install setuptools -U
+
+echo "-----------------------------------------------------------------"
+echo "Installing CKAN and its Python dependencies..."
+
+cd .. # CircleCI starts inside ckanext-geodatagov folder
+pwd
+ls -la
+
+git clone https://github.com/ckan/ckan
+cd ckan
+if [ $CKANVERSION == '2.8' ]
+then
+	git checkout 2.8
+elif [ $CKANVERSION == '2.3' ]
```

Comment:
For 2.3, we actually want the data.gov fork, https://github.com/GSA/ckan/tree/datagov

Reply:
fixed